docs/learn/learn-polkadot-opengov.md

```diff
@@ -306,10 +306,10 @@ include:
   Period (note that more requirements must be met to enter the Decision Period).
 - **Preparation Period**: the minimum amount of voting time needed before entering the Decision
   Period (given capacity and deposit are met).
-- **Decision Period**: the maximum time to approve a proposal. The proposal will be accepted if
-  approved by the end of the period.
-- **Confirmation Period**: the minimum amount of time (within the Decision Period) the approval and
-  support criteria must hold before the proposal is approved and moved to the enactment period.
+- **Decision Period**: the time interval during which a proposal's outcome can be decided. 
+- **Confirmation Period**: the minimum amount of time the approval and
+  support criteria must hold before the proposal is approved and moved to the enactment period. The
+  confirmation period should start before the end of decision period.
```

Comment:
```suggestion
:::info Edge Case

A referendum can enter the confirmation period one block before the decision period ends. In this scenario, the referendum will pass if it satisfies approval and support thresholds for the minimum confirmation period (track-dependent).

:::
```

Reply:
Sounds good. I'll make sure to add it. Although, I don't think it was your intention to remove line 312 right?